app/services/manage_donation.rb

```diff
@@ -0,0 +1,9 @@
+class ManageDonation < ManageAction
+  private
+  def queue_message
+    {
+        type: 'donation',
+        params: @params
```

Comment:
I suspect these need to be constructed somewhere, though, since the fields we get back from BrainTree response objects or what we have locally don't correspond directly to the ActionKit api.